script/unit-tests.ts

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env ts-node
+
+import * as Path from 'path'
+import { spawnSync, SpawnSyncOptions } from 'child_process'
+
+const environmentVariables = {
+  // setting commit information so that tests don't need to rely on global config
+  GIT_AUTHOR_NAME: 'Joe Bloggs',
+  GIT_AUTHOR_EMAIL: 'joe.bloggs@somewhere.com',
+  GIT_COMMITTER_NAME: 'Joe Bloggs',
+  GIT_COMMITTER_EMAIL: 'joe.bloggs@somewhere.com',
+  // signalling to dugite to use the bundled Git environment
+  TEST_ENV: '1',
+  // ensuring Electron doesn't attach to the current console session (Windows only)
+  ELECTRON_NO_ATTACH_CONSOLE: '1',
+}
+
+const env = Object.assign({}, process.env, environmentVariables)
+
+const root = Path.dirname(__dirname)
```

Comment:
Could we add a little comment explaining this and perhaps consider renaming it to something less generic?